test/e2e/run_dns_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("Podman run dns", func() {
 		session := podmanTest.Podman([]string{"run", "--hostname=foobar", ALPINE, "cat", "/etc/hostname"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
-		Expect(session.OutputToString()).To(Equal("foobar"))
+		Expect(session.OutputToString()).To(Equal("foobar\n"))
```

Comment:
I did not realize that this test existed, otherwise I would have done this in my PR.